<launch>
    <node pkg="actuator" type="actuator_master.py" name="act_master" output="screen" />
</launch>